apparmor.d/profiles-s-z/tlp

```diff
@@ -0,0 +1,117 @@
+# apparmor.d - Full set of apparmor profiles
+# Copyright (C) 2021-2024 Alexandre Pujol <alexandre@pujol.io>
+# Copyright (C) 2024 Barmogund <set508@proton.me>
+# SPDX-License-Identifier: GPL-2.0-only
+
+abi <abi/4.0>,
+
+include <tunables/global>
+
+@{exec_path} = @{bin}/tlp
+profile tlp @{exec_path} flags=(attach_disconnected) {
+  include <abstractions/base>
+  include <abstractions/bus-system>
+  include <abstractions/disks-read>
+  include <abstractions/graphics>
+  include <abstractions/bus/org.freedesktop.PolicyKit1>
+  include <abstractions/nameservice-strict>
+  include <abstractions/perl>
+
+  capability dac_read_search,
+  capability net_admin,
+  capability sys_nice,
+  capability sys_rawio,
+  capability sys_tty_config,
+
+  network netlink raw,
+
+  ptrace read peer=unconfined,
+
+  @{exec_path} mr,
+
+  @{bin}/systemctl              rix,
+  @{bin}/logger                 rix,
+  @{sh_path}                    rix,
+  @{bin}/cp                     rix,
+  @{bin}/chmod                  rix,
+  @{bin}/flock                  rix,
+  @{bin}/sort                   rix,
+  @{bin}/head                   rix,
+  @{bin}/mktemp                 rix,
+  @{bin}/readlink               rix,
+  @{bin}/tr                     rix,
+  @{bin}/ethtool                rix,
+  @{bin}/grep                   rix,
+  @{bin}/touch                  rix,
+  @{bin}/cat                    rix,
+  @{bin}/rm                     rix,
+  @{bin}/id                     rPx,
+  @{bin}/iw                     rPx,
+  @{bin}/hdparm                 rPx,
+  @{bin}/uname                  rpx,
+  @{bin}/udevadm rCx            -> udevadm,
+  /usr/share/tlp/tlp-readconfs  rix,
+
+  / r,
+
+  /etc/tlp.d/ r,
+  /etc/tlp.d/** rw,
+  /etc/tlp.conf rw,
+  /etc/udev/udev.conf r,
```

Comment:
Since udevadm is not part of the profile. This rule might not be needed anymore.